.github/workflows/build.yml

```diff
@@ -0,0 +1,73 @@
+name: CI-Build
+
+on:
+    push:
+      paths:
+        - 'src/**'
+        - 'include/**'
+        - 'meson.build'
+    pull_request:
+      paths:
+        - 'src/**'
+        - 'include/**'
+        - 'meson.build'
+
+jobs:
+  build-cpp:
+    runs-on: ubuntu-latest
+    
+    defaults:
+      run:
+        working-directory: ./libfakedevice-cpp
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Install dependencies
+      run: |
+        sudo apt update
+        sudo apt install meson clang
+
+    - name: Build C++ project
+      run: ./autorun.sh
+
+  build-c:
+    runs-on: ubuntu-latest
+
+    defaults:
+      run:
+        working-directory: ./libfakedevice-c
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Install dependencies
+      run: |
+        sudo apt update
+        sudo apt install meson clang
+        sudo meson wrap update-db
```

Comment:
Is this line a leftover or is it necessary?

Reply:
Strange, I had removed it! Anyway, it is not necessary.